apps/issuer/v1_api_urls.py

```diff
@@ -1,15 +1,15 @@
 from django.conf.urls import url
 
-from issuer.api import (IssuerList, IssuerDetail, IssuerBadgeClassList, BadgeClassDetail, BadgeInstanceList,
-                        BadgeInstanceDetail, IssuerBadgeInstanceList, AllBadgeClassesList, BatchAssertionsIssue,
+from issuer.api import (IssuerDetail, IssuerBadgeClassList, BadgeClassDetail, BadgeInstanceList,  # IssuerList,
+                        BadgeInstanceDetail, IssuerBadgeInstanceList, BatchAssertionsIssue,  # AllBadgeClassesList,
                         TimestampedBadgeInstanceList, BatchSignAssertions)
 from issuer.api_v1 import FindBadgeClassDetail # , IssuerStaffList, IssuerStaffConfirm
 
 urlpatterns = [
-    url(r'^all-badges$', AllBadgeClassesList.as_view(), name='v1_api_issuer_all_badges_list'),
+    # url(r'^all-badges$', AllBadgeClassesList.as_view(), name='v1_api_issuer_all_badges_list'),
     url(r'^all-badges/find$', FindBadgeClassDetail.as_view(), name='v1_api_find_badgeclass_by_identifier'),
```

Comment:
This one and `issuer/slug` are already replaced by graphql so I think they can go too